languages/common-lisp/exercises/concept/lillys-lasagna-leftovers/lillys-lasagna-leftovers-test.lisp

```diff
@@ -0,0 +1,51 @@
+;; Ensures that lillys-lasagna-leftovers.lisp and the testing library are always loaded
+(eval-when (:compile-toplevel :load-toplevel :execute)
+  (load "lillys-lasagna-leftovers")
+  (ql:quickload :fiveam))
+
+;; Defines the testing package with symbols from lillys-lasagna-leftovers and FiveAM in scope
+;; The `run-tests` function is exported for use by both the user and test-runner
+(defpackage :lillys-lasagna-leftovers-test
+  (:use :cl :fiveam :lillys-lasagna-leftovers)
+  (:export :run-tests))
+
+;; Enter the testing package
+(in-package :lillys-lasagna-leftovers-test)
+
+;; Define and enter a new FiveAM test-suite
+(def-suite lillys-lasagna-leftovers-suite)
+(in-suite lillys-lasagna-leftovers-suite)
+
+(test preparation-time
+  (is (= 0 (preparation-time)))
+  (is (= 38 (preparation-time 'sauce 'cheese)))
+  (is (= 95 (preparation-time 'sauce 'cheese 'left-handed-macaroni 'cheese 'cheese))))
+
+(test remaining-minutes-in-oven
+  (is (= 337 (remaining-minutes-in-oven)))
+  (is (= 337 (remaining-minutes-in-oven :normal)))
+  (is (= 237 (remaining-minutes-in-oven :shorter)))
+  (is (= 437 (remaining-minutes-in-oven :longer)))
+  (is (= 137 (remaining-minutes-in-oven :very-short)))
+  (is (= 537 (remaining-minutes-in-oven :very-long)))
+  (is (= 0   (remaining-minutes-in-oven nil))))
+
+(test splitting-leftovers
```

Comment:
I think I make make the test-runner treat whole test-blocks as a unit, so we might want to start breaking up assertions into more `test` blocks. This one could probably be at least 3 (just going off of your whitespace).

Reply:
OK, will split it up some more.